mtp_noms_ops/templates/security/auto_accept_rule_list.html

```diff
@@ -0,0 +1,105 @@
+{% extends 'base.html' %}
+{% load i18n %}
+{% load mtp_common %}
+{% load security %}
+
+{% block page_title %}{{ view.title }} – {{ block.super }}{% endblock %}
+
+
+{% block inner_content %}
+  <nav class="moj-sub-navigation" aria-label="{% trans 'Sub navigation' %}">
+    <ul class="moj-sub-navigation__list">
+      <li class="moj-sub-navigation__item">
+        <a class="moj-sub-navigation__link" href="{% url 'security:check_list' %}">{% trans 'Credits to action' %}</a>
+      </li>
+      <li class="moj-sub-navigation__item">
+        <a class="moj-sub-navigation__link" href="{% url 'security:my_check_list' %}">{% trans 'My list' %} ({{ form.my_list_count }})</a>
+      </li>
+      <li class="moj-sub-navigation__item">
+        <a class="moj-sub-navigation__link" href="{% url 'security:credits_history' %}" >{% trans 'Decision history' %}</a>
+      </li>
+      <li class="moj-sub-navigation__item">
+        <a class="moj-sub-navigation__link" aria-current="page" href="{% url 'security:auto_accept_rule_list' %}" >{% trans 'Auto Accepts' %}</a>
+      </li>
+    </ul>
+  </nav>
+  <header>
+    <h1 class="heading-xlarge">{{ view.title }}</h1>
+  </header>
+
+  {% include 'mtp_common/includes/message_box.html' %}
+
+  <p class="font-large">
+    {% blocktrans trimmed count count=form.total_count with number=form.total_count|separate_thousands %}
+    {{ number }} auto accept rule
+    {% plural %}
+    {{ number }} auto accept rules
+    {% endblocktrans %}
+  </p>
+
+  <div class="mtp-results-list mtp-check-list">
+    <table class="mtp-table table-font-xsmall">
+      <thead>
+        <tr>
+          <th>{% trans 'Debit card (from)' %}</th>
```

Comment:
could you please add `scope="col"` to these table headers – it's something that was picked up before RE accessibility but is easy to miss. don't worry about `sortable_cell`, i added scope to it in https://github.com/ministryofjustice/money-to-prisoners-common/pull/405/